src/createServerClient.ts

```diff
@@ -170,11 +163,11 @@ export function createServerClient<
         ...(options?.cookieOptions?.name
           ? { storageKey: options.cookieOptions.name }
           : null),
-        ...options?.auth,
         flowType: "pkce",
         autoRefreshToken: false,
         detectSessionInUrl: false,
         persistSession: true,
+        ...options?.auth,
```

Comment:
It would, currently is it not possible

Reply:
You're right, it would make it configurable. I'm still not sure it'll fix the concern around a service role client though. Let's complete discussing the issue on the thread and circle back to this PR if needed